WORKFLOW.md

```diff
@@ -59,3 +59,74 @@ PR and proposal was approved by "vast majority".
 
 You are welcome to propose changes to this workflow by opening an
 [issue](https://github.com/fortran-lang/stdlib/issues).
+
+
+## Build system
+
+This project supports two build systems right now, CMake and make.
+Eventually, stdlib will be using the Fortran package manager
+([fpm](https://github.com/fortran-lang/fpm)) as build system as well.
+The effort of supporting fpm is tracked in issue
+[#279](https://github.com/fortran-lang/stdlib/issues/279).
+
+
+### CMake build files
+
+The build files for CMake allow both in-tree, *i.e.* build artifacts share
+the same tree as the source files, and out-of-tree builds, *i.e.* build artifacts
+exist in a separate directory tree.
+Both build types are explicitly supported and tested, the latter strategy
+is recommended for local development.
+
+Sources for the main library target are added in ``src/CMakeLists.txt``
+relative to their the library target, *i.e.* no absolute paths are required.
+
+To add tests the macro ``ADDTEST`` should be used in instead of the CMake function
+``add_test``, it hides creation of the executable target, linking against the
```

Comment:
It is not clear  to what the "it" refers to.

Reply:
```suggestion
 ``add_test``, the macro hides creation of the executable target, linking against the 
```